site/views/blogs/ESL v5.0.0 Migartion Guide.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: content
+name: ESL v5.0.0 Migartion Guide
+title: Migrating from ESL v4.*.* to ESL v5.0.0
+tags: [news, blogs, draft]
+date: 2024-11-31
+---
+
+The ESL version 5.0.0 have just been released and it comes with a lot of new features and improvements. 
+This guide will help you to migrate your existing ESL v4.\*.* project to ESL v5.0.0.
+
+---
+
+## Preparation
+We recommend you to consider using ESL ESLint plugin to check your codebase for ESL v5.0.0 compatibility.
+It will help you to find and fix most of the issues before you start the migration process.
+The points not marked with `🔧` in the breaking changes section are covered by the ESLint plugin checks and could be fixed automatically.
+
+---
+
+## Breaking Changes
+
+- ### Browser Support
+  ESL v5.0.0 no longer supports IE11 and ES5 target. 
+  ESL UI site renderer and ESL polyfills no longer support Edge old versions and ES6 polyfils.
+  In case by some reason you still need to support IE11 or ES5 target you should use proper transpilation and polyfilling tools on your side.
```

Comment:
```suggestion
  In case for some reason you still need to support IE11 or ES5 target you should use proper transpilation and polyfilling tools on your side.
```